website/docs/reference/resource-configs/databricks-configs.md

```diff
@@ -182,6 +182,9 @@ insert overwrite table analytics.databricks_incremental
 </TabItem>
 </Tabs>
 
+#### Usage Notes
+* This strategy is not currently compatible with SQL Warehouses
```

Comment:
1. changed it to be a "callout"
2. explicitly stated where is does work
3. perhaps also consider moving this to be directly under the section header at L101, instead of "buried" at the end?


```suggestion
:::caution
This strategy is only supported for General Purpose Clusters, not SQL Warehouses
:::
```